frontend/www/js/omegaup/components/arena/RunSubmit.vue

```diff
@@ -65,39 +74,74 @@ import { omegaup } from '../../omegaup';
 import * as ui from '../../ui';
 import T from '../../lang';
 import arena_CodeView from './CodeView.vue';
+import omegaup_Countdown from '../Countdown.vue';
 
 @Component({
   components: {
     'omegaup-arena-code-view': arena_CodeView,
+    'omegaup-countdown': omegaup_Countdown,
   },
 })
 export default class ArenaRunSubmit extends Vue {
   @Ref() inputFile!: HTMLInputElement;
-  @Prop() languages!: omegaup.Languages;
-  @Prop({ default: 0 }) submissionGapSecondsRemaining!: number;
+  @Prop() languages!: string[];
+  @Prop({ default: Date.now() }) nextSubmissionTimestamp!: Date;
```

Comment:
```suggestion
  @Prop({ default: new Date() }) nextSubmissionTimestamp!: Date;
```

[`Date.now()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) regresa un número, no una fecha. (Esto es lo que está causando las fallas en las pruebas).

Reply:
Cierto, no lo consideré.